.github/workflows/build.yml

```diff
@@ -30,6 +28,33 @@ jobs:
       - run: npm run lint
       - run: npm test
 
+  e2e:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Check and restore cache of playwright
+        uses: actions/cache@v3
+        id: playwright-cache
+        with:
+          path: |
+            ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ hashFiles('**/package-lock.json') }}
+      - run: npm install
+      - run: npx playwright install --with-deps chromium
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+      - run: npm run build
+      - run: chmod 777 dist/cli.js
```

Comment:
I'm wondering why I need to `chmod 777 dist/cli.js` before running `dist/cli.js`.
If I don't do this, I get a permission error.
It's might be a bug, but I believe that should be clearly describe and fix in another issue/pull request.

Reply:
TypeScript won't set execute permissions. I believe npm will set execute permissions to the files in the ["bin" section](https://github.com/unvt/charites/blob/main/package.json#L5-L7) when running npm package (or maybe npm install?). Here we can do `chmod +x dist/cli.js` or run it via node like `node dist/cli.js`. I don't think it's a big problem.